test/util/test_helpers.py

```diff
@@ -171,39 +171,35 @@ def update_policy_from_string(self, policy):
     return output
 
 
-# *************************************************
-# *************** UNIT TESTS HELPERS **************
-# *************************************************
+# *************** CREDENTIALS ***************
 
-def validate_netrc_contents(self):
-    with open('path/to/netrc', 'r') as netrc:
-        lines = netrc.readlines()
-        self.assertEquals(lines[0].strip(), "machine https://someurl")
-        self.assertEquals(lines[1].strip(), "login somelogin")
-        self.assertEquals(lines[2].strip(), "password somepass")
+def create_cred_store():
+    cred_store, _ = CredentialStoreFactory().create_credential_store()
+    return cred_store
 
 
 def get_credentials() -> CredentialsData:
     try:
-        cred_store, _ = CredentialStoreFactory().create_credential_store()
+        cred_store = create_cred_store()
         conjurrc = ConjurrcData.load_from_file()
```

Comment:
Don't think this function should be called load_from_file since we have from_file and from_keyring flows